sb_vision/native/apriltag/detector.py

```diff
@@ -1,14 +1,19 @@
 """AprilTag detector wrapper."""
 
+from typing import TYPE_CHECKING, Any, Iterator, Tuple
+
 from PIL import Image
 
 from ._apriltag import ffi, lib
 
+if TYPE_CHECKING:
```

Comment:
It looks like there are a few places we're importing for only the sake of type definitions. Is this conditionally really necessary, or is it something overlooked in other locations

Reply:
Does 8cf9295 help?

Reply:
For clarity: in general I think we should do plain imports of types which are needed for type checking, however the constant is useful when either that would result in the import being circular or the types being imported aren't actually importable at runtime (as in this case).